pkg/validation/limits.go

```diff
@@ -170,7 +170,7 @@ type Limits struct {
 	BlockedQueries []*validation.BlockedQuery `yaml:"blocked_queries,omitempty" json:"blocked_queries,omitempty"`
 
 	RequiredLabels       []string `yaml:"required_label_matchers,omitempty" json:"required_label_matchers,omitempty" doc:"description=Define a list of required selector labels."`
-	RequiredNumberLabels int      `yaml:"required_number_label_matchers,omitempty" json:"required_number_label_matchers,omitempty" doc:"description=Minimum number of label matchers a query should contain."`
+	RequiredNumberLabels int      `yaml:"minimum_labels_number,omitempty" json:"required_number_label_matchers,omitempty" doc:"description=Minimum number of label matchers a query should contain."`
```

Comment:
```suggestion
	RequiredNumberLabels int      `yaml:"minimum_labels_number,omitempty" json:"minimum_labels_number,omitempty" doc:"description=Minimum number of label matchers a query should contain."`
```

Reply:
Thank buddy! I need to stop working on two things at the same time 😅

Reply:
We didn't rename it for `required_label_matchers` either :see_no_evil:

Reply:
Done :)